CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.1)
+cmake_minimum_required(VERSION 3.2)
```

Comment:
Seems fine, we can probably even go higher if that makes things easier, given most people build this in docker images anyway.